README.md

```diff
@@ -1 +1,55 @@
-# Capstone-Project1
\ No newline at end of file
+# Confrence Summit
+
+This is the capstone project from the first module of Microverse Program.This is a website built by myself to serve clients who wants to confrence a summit and give information about its previous confrence summits happened worldwide.
+
+## Built With
+
+Html/Css
+Javascript
+Linters
+
+## Getting Started
+
+My project is responsive and works on mobile and desktop using @Media queries.
+
+## Setup
+
+To get a local copy up and running follow these simple example steps.
+
+To get access to this repository, move to "capstone-project" repository, and then either use [ssh] or [https] to clone onto your local machine. To clone checkout this documentation on github (https://docs.github.com/en/repositories/creating-and-managing-repositories/cloning-a-repository)
+
+## Live Demo
+
+https://jusha25.github.io/Capstone-Project1/
+
```

Comment:
- [ ] the live demo link is not working as expected please fix this before submitting it for another review.